autogpt_platform/frontend/src/lib/autogpt-server-api/types.ts

```diff
@@ -500,11 +500,11 @@ export enum BlockUIType {
   WEBHOOK = "Webhook",
   WEBHOOK_MANUAL = "Webhook (manual)",
   AGENT = "Agent",
+  AI = "AI",
```

Comment:
I don't see this being used anywhere, was this included by accident?

Reply:
It's just a new type block that is introduced in the backend but not introduced here.